lib/src/model.dart

```diff
@@ -477,6 +482,24 @@ abstract class ReportSectionId {
   static const dependency = 'dependency';
   static const documentation = 'documentation';
   static const platform = 'platform';
+  static const nullSafety = 'null-safety';
+}
+
+enum ReportStatus {
```

Comment:
As it becomes part of the public API, let's review the naming a bit, please consider:
- `miss`, `failed` (instead of `bad`)?
- `mixed`, `partial` (instead `soso`)?
- `pass`, `ok` (instead of `good`)?

Reply:
Yeah - I guess these sound more professional ;)

Fixed